sdk/eventhub/Azure.Messaging.EventHubs/src/RetryMode.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.EventHubs
+{
+    /// <summary>
+    ///   The type of approach to apply when calculating the delay
+    ///   between retry attempts.
+    /// </summary>
+    ///
+    public enum RetryMode
```

Comment:
Technically we could, but I don't feel that's the way to go.   Since `Core` doesn't provide a non-HTTP pipeline implementation, there's no corresponding policy to use.  I don't want to have the `Core` enum evolve to change offerings that Event Hubs doesn't support - or remove one that does.   Since we're talking just an enum, I'm opting for a bit of repetition for the safety of being stable and in control.